### PR TITLE
DOC:2041: Added TINY-9148 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC:2041

Changes:
* Added TINY-9148 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -71,6 +71,11 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Closing a dialog would scroll down the document in Safari. #TINY-9148
```
Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:74]:
```suggestion
=== Closing a dialog would scroll down the document in Safari. 
//#TINY-9148
```

**Review by kemister85 — COMMENTED** [review]
commenting out TINY-ref

**Review by yacodes — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -71,6 +71,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Closing a dialog would scroll down the document in Safari. 
+//#TINY-9148
+
+In Safari, when closing a dialog, a bug caused the browser to select the a element and scroll down. As a consequence, the editor would scroll to the bottom after the dialog was closed.
+
+To fix this issue, {productname} now forces the active element to blur in Safari. By doing so, no element will be selected after closing the dialog, preventing the unintended scrolling behavior.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:79]:
```suggestion
=== Closing a dialog would scroll down the document in Safari on macOS. 
//#TINY-9148

In Safari running on macOS, when a dialog was closed by clicking outside the dialog, a bug caused the browser to scroll the active element and scroll down.

This caused the browser to scroll the viewport to the bottom of the dialog after the dialog closed.

This scrolling did not occur if the dialog was closed by pressing the dialog close control or by typing the *Esc* key.

To fix this issue, {productname} now forces the active element to blur when Safari is the host browser. By doing so, no element is selected after closing the dialog, preventing the unintended scrolling behavior.
```

Edits for specificity and added note re the specific action that produces the behaviour.

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 3]
```diff
@@ -71,6 +71,16 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Closing a dialog would scroll down the document in Safari on macOS. 
+//#TINY-9148
+
+In Safari running on macOS, when a dialog was closed by clicking outside the dialog, a bug caused the browser to scroll the active element and scroll down.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:77]:
```suggestion
In Safari running on macOS, when a dialog was closed by clicking outside the dialog, a bug caused the browser to select the active element and scroll down.
```

Fixing my own typo: s/scroll/select/.

Sorry.

**Pull request merged**